app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Users
```

Comment:
why do you need to wrap it in a module?

Reply:
when "devise" generate his controller it was
class Users::RegistrationsController < Devise::RegistrationsController
but rubocop didn't allow me use shortcut method, he asked about nested style.